src/qt/guiutil.cpp

```diff
@@ -774,6 +774,18 @@ QString formatBytes(uint64_t bytes)
     return QObject::tr("%1 GB").arg(bytes / 1'000'000'000);
 }
 
+QString formatBps(uint64_t bytes)
+{
+    if (bytes < 1'000)
+        return QObject::tr("%1 B/s").arg(bytes);
```

Comment:
should add Translator comments

see: https://doc.qt.io/qt-5/i18n-source-translation.html#translator-comments

Reply:
A translator [comment](https://github.com/bitcoin-core/gui/pull/305#discussion_r624665523) is the easiest way to let translators know what "bps" is.